.ci/Jenkinsfile

```
@@ -581,11 +585,21 @@ def generateFunctionalTestStep(Map args = [:]){
                 runId,
                 "-t destroy --extra-vars=\"nodeLabel=${platform} nodeUser=${machine.username} nodeImage=${machine.image} nodeInstanceType=${machine.instance_type}\"")
         }
+        withEnv([
+          "ARCHITECTURE=${goArch}",
+          "CUCUMBER_REPORTS_PATH=${BASE_DIR}/outputs/${testRunner.ip}",
+          "PLATFORM=${platform}",
+          "SUITE=${suite}",
+          "TAGS=${tags}",
+        ]){
+          dockerLogin(secret: "${DOCKER_ELASTIC_SECRET}", registry: "${DOCKER_REGISTRY}")
+          sh ".ci/scripts/generate-cucumber-reports.sh"
```

NIT, to help with debugging in the CI/OTEL, you can use the `label`

```suggestion
          sh(script: ".ci/scripts/generate-cucumber-reports.sh", label: "generate-cucumber-reports.sh")
```